backend/README.md

```diff
@@ -53,28 +55,32 @@ Here's a useful psql command cheat sheet: [https://gist.github.com/Kartones/dd3f
 
 ## API Endpoints
 
-### Basic Search API
+### Clinical Studies Search API
 #### Parameters
 
-The end point for Basic Search API is:
-<http://127.0.0.1:8000/ctgov/api/basic_search>
+<span style="color:red">Please note api endpoint name is changed to "search_studies" to be more generic.</span>
+
+The end point for Clinical Studies Search API is:
+<http://127.0.0.1:8000/ctgov/api/search_studies>
+
+<span style="color:red">Please note the change in api parameters. If metadata is required, pass "metadata_required"="True"; if metadata is not required (for example, during pagination) pass empty string as "metadata_required="".</span>
 
-Example query located in`database/examples/basic_search_query.json`:
+Example query located in `database/examples/basic_search_query.json`:
 
 	{
 	    "status":"Recruiting",
-	    "condition":"",
-	    "other_terms":"POETYK",
-	    "country":"",
-	    "intervention":"",
-	    "target":"",
-	    "nct_id":"",
-	    "eligibility_criteria":"",
+		"condition":"",
```

Comment:
Fix indentation here. Everything should be indented the same amount.

Reply:
I can fix this.